src/main/java/com/betadevels/onlineshopping/action/subscription/UpdateSubscriptionStatusAction.java

```diff
@@ -0,0 +1,67 @@
+package com.betadevels.onlineshopping.action.subscription;
+
+import com.betadevels.onlineshopping.action.Action;
+import com.betadevels.onlineshopping.db.hibernate.SubscriptionHibernateDAO;
+import com.betadevels.onlineshopping.enumerations.SubscriptionStatus;
+import com.betadevels.onlineshopping.exceptions.InternalErrorException;
+import com.betadevels.onlineshopping.models.Subscription;
+import com.betadevels.onlineshopping.payload.request.subscription.UpdateSubscriptionStatusRequest;
+import com.betadevels.onlineshopping.payload.response.subscription.SubscriptionListResponse;
+import com.betadevels.onlineshopping.payload.response.subscription.SubscriptionResponse;
+import com.betadevels.onlineshopping.util.HibernateUtil;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import org.hibernate.HibernateException;
+import org.modelmapper.ModelMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Created by girijagodbole on 4/16/17.
+ */
+public class UpdateSubscriptionStatusAction implements Action<SubscriptionListResponse> {
```

Comment:
Why do we need a separate API to update the status of a subscription? @prathyushark already has the update subscription API. Add the status update there itself.